quickstart.mdx

```diff
@@ -1,29 +1,31 @@
 ---
 title: 'Quickstart'
-description: 'Everything you need to start using Plain.'
+description: 'Everything you need to know to start supporting your customers with Plain.'
 ---
 
 <Frame>![Plain.com](/public/images/quickstart.png)</Frame>
 
-Plain is the support platform for B2B technical teams. Everything you need to help your customers in a modern, fast, and opinionated platform.
+Plain is the fastest, most powerful support platform for B2B startups. It's got everything you need to help your customers in a modern, collaborative, and consolidated platform.
```

Comment:
Is saying 'B2B startups' here a bit self limiting? I worry that someone bigger might be put off. Our headline on the marketing site says 'B2B teams'